CodeU_Assignement_3/src/codeu_assignement_3/SimpleDictionary.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */

```

Comment:
It tells you in the comment how to change the templates and remove that boilerplate.